app.py

```diff
@@ -103,6 +116,10 @@ def stats(api, memory_used_gauge, memory_total_gauge, network_up_gauge, network_
     start_http_server(9999)
     set_static_info(api)
 
+    volume_states = ["normal","attention","error"]
```

Comment:
Let's move these higher up, they are kind of implicitly global since they are set just before the function call.